runtime/cfdumper/main.c

```diff
@@ -1437,12 +1437,20 @@ static void printDisassembledMethod(J9CfrClassFile* classfile, J9CfrMethod* meth
 			NEXT_U8_ENDIAN(bigEndian, bc, bcIndex);
 			if(wide)
 			{
-				j9tty_printf( PORTLIB, "%s ", sunJavaBCNames[bc]);
+				j9tty_printf( PORTLIB, "%s", sunJavaBCNames[bc]);
 			}
 			else
 			{
-				j9tty_printf( PORTLIB, "%5i %s ", pc, sunJavaBCNames[bc]);
+				j9tty_printf( PORTLIB, "%5i %s", pc, sunJavaBCNames[bc]);
 			}
+
+			if(0 == strcmp(sunJavaBCNames[bc], "JBunimplemented")) {
```

Comment:
This file does not follow the current coding standards, but we should be consistent with what is already there.

the `{` should be on the next line